packages/@withstudiocms/internal_helpers/src/debug-provider/infra/bun-package-manager.ts

```diff
@@ -0,0 +1,8 @@
+import type { PackageManager } from '../definitions.js';
+
+/**
+ * Represents the Bun package manager.
+ */
+export class BunPackageManager implements PackageManager {
```

Comment:
Why are these classes? What are these doing? Why isn't this just one class you pass a "packageManager" string argument into? Idk I'm very against classes generally, in this case they may just be over-engineering